PowerEditor/src/ScitillaComponent/ScintillaEditView.h

```diff
@@ -632,6 +633,9 @@ friend class Finder;
     void setTabSettings(Lang *lang);
 	bool isWrapRestoreNeeded() const {return _wrapRestoreNeeded;};
 	void setWrapRestoreNeeded(bool isWrapRestoredNeeded) {_wrapRestoreNeeded = isWrapRestoredNeeded;};
+	bool isPositionRestoreNeeded() const {return _positionRestoreNeeded;};
+	void setPositionRestoreNeeded(bool isPositionRestoredNeeded) { _positionRestoreNeeded = isPositionRestoredNeeded;};
+

```

Comment:
Both getter and setter are not necessary here since `_positionRestoreNeeded` is only used internally (in the object).

Reply:
Ok ! I've removed the setter.